### PR TITLE
KOGITO-9837: Reuse components migrated from kogito-apps

### PR DESCRIPTION
**Pull request by fantonangeli**
**Jira:** https://issues.redhat.com/browse/KOGITO-9837

**Description:**
This PR is to reuse the code from runtime-* packages in  "sonataflow-deployment-webapp"

### CODE REVIEW

File: packages/runtime-tools-gateway-api/src/gatewayApi/apis.tsx [hunk 1]
```diff
@@ -481,19 +483,19 @@ const doTriggerCloudEvent = (event: CloudEventRequest, baseUrl: string, proxyEnd
   const url = `${baseUrl}${event.endpoint.startsWith("/") ? "" : "/"}${event.endpoint}`;
 
   return axios.request({
-    url: proxyEndpoint,
+    url: proxyEndpoint || url,
     method: event.method,
     data: cloudEvent,
     headers: {
-      "Target-Url": url,
+      ...(proxyEndpoint ? { "Target-Url": baseUrl } : {}),
```
Comment (caponetto) [inline packages/runtime-tools-gateway-api/src/gatewayApi/apis.tsx]:
```suggestion
      ...(proxyEndpoint ? { "Target-Url": url } : {}),
```
I believe you meant `url` here

**Comment by caponetto** [comment]
Hey @paulovmr, could you please take a look at this PR when you have some time? Thanks!

**Comment by caponetto** [comment]
Hey @fantonangeli, I'm seeing an infinite loading on web tools. `main` looks fine. Could you please check?

https://github.com/kiegroup/kie-tools/assets/638737/155d341e-7a85-4eee-b0cd-fec63a58d54c

**Comment by fantonangeli** [comment]
> Hey @fantonangeli, I'm seeing an infinite loading on web tools. `main` looks fine. Could you please check?
> Screen.Recording.2023-10-04.at.09.48.52.mov

Hey @caponetto , thanks for seeing this bug.
I pushed the fix: https://github.com/kiegroup/kie-tools/pull/1995/commits/f6a1c99edff1f75ca93fd727fbb397aad51b3317

**Review by caponetto — APPROVED** [review]

**Review by paulovmr — APPROVED** [review]

**Review by tomasdavidorg — APPROVED** [review]

**Pull request merged**